onboard/catkin_ws/src/cv/scripts/utils.py

```diff
@@ -53,3 +54,298 @@ def visualize_detections(self, frame, detections):
             self.rectangle(frame_copy, bbox)
 
         return frame_copy
+
+
+class ImageTools(object):
```

Comment:
Can this be its own file, rather than throwing it in `utils.py`?

Reply:
Moved `ImageTools` class to `image_tools.py` and changed imports for class in other files.